All project code/index.js

```diff
@@ -196,6 +196,207 @@ app.get("/profile", (req, res) => {
     }});
 });
 
+
+app.get('/visit', (req, res) => {
+    res.render("pages/visit");
```

Comment:
What is this doing? The visit page needs parameters correct? If so a user should not be able to just render this page like this without any parameters or queries

Reply:
Good catch, it isn't doing anything and is probably incorrect to include. I'll remove it.